### DISCUSSION TITLE
Should the cask-font tap be indexed in brew.sh search?

### DISCUSSION THREAD

**Discussion by jasonkarns** [Homebrew/discussions#5347] [Everyday usage]
### Output of `brew config`

```shell
na
```

### Output of `brew doctor`

```shell
na
```

### Description of issue

Searching brew.sh for a font (font-*) doesn't seem to find anything from the cask-font tap. It seems that would be a useful tap to index (particularly because it's "official"?)

**Comment by jasonkarns** [comment]
Perhaps this would be a PR to https://github.com/Homebrew/formulae.brew.sh to add cask-font to the site and thus also the algolia index?

Though I know how much automation is in that repo so this would be a big change.

**Comment by Bo98** [comment]
We're thinking about merging homebrew-cask-fonts into homebrew-cask so if/when that happens will end up fixing this.

**Reply by jasonkarns** [reply]
Cool! Is that decided or still just under consideration?
